docs/src/integrate/nodejs-api.md

```diff
@@ -658,6 +662,7 @@ The information available for each linting message is:
 * `fatal` - usually omitted, but will be set to true if there's a parsing error (not related to a rule).
 * `line` - the line on which the error occurred.
 * `message` - the message that should be output.
+* `messageId` - the ID of the message used to generate the message (this property is omitted if the rule does not use message ids).
```

Comment:
Thanks! could we agree on capitalizing `ID` and `IDs` in the descriptions if that's the common way of writing? `id` and `ids` would be also fine, I'd just prefer to stay consistent.

Reply:
I use the capitalized variant because it was used for `ruleId` but forgot to update the others. Also updated one for `messageId` in `errors` which used "Id".